webauthn/webauthn.py

```diff
@@ -85,9 +85,11 @@ class WebAuthnUserDataMissing(Exception):
 class WebAuthnMakeCredentialOptions(object):
 
     _attestation_forms = {'none', 'indirect', 'direct'}
+    _userVerification = {'required','preferred','discouraged'}
 
     def __init__(self, challenge, rp_name, rp_id, user_id, username,
-                 display_name, icon_url, timeout=60000, attestation='direct'):
+                 display_name, icon_url, timeout=60000, attestation='direct',
+                 userVerification=None):
```

Comment:
Instead of `None` here, can we default to one of the options? Maybe `preferred`?

Reply:
The use of None instead of a "normal" value is to prevent adding the authenticatorSelection value to the registration_dict if not provided.
I think there is a difference between not setting the value (hence letting the operating system decinding the behaviour) and forcing to the standard value. A change in the standard will be pushed directly in usage if we use None ; if not, we will have to patch this function with the new standard value.

The final behaviour is probably the same.

Reply:
Fair enough. Let's go ahead and leave the default value as `None`, then.

Reply:
I dont want to force in any way, you are the owner of the project!
I juste want to contribute in the best way possible to this awsome project.
PS: And by the way, it's my first (real) open-source contribution (not taking into account my own projects, of course).